doc/source/google/step-zero-gcp.rst

```diff
@@ -84,7 +84,21 @@ your google cloud account.
    * ``--zone`` specifies the data center zone where your cluster will be created.
      You can pick something from `this list
      <https://cloud.google.com/compute/docs/regions-zones/#available>`_
-     that is not too far away from your users.
+     that is not too far away from your users.                   
+   *  .. note::
+         
+         A region is a geographical location used to host resources, comprising one or more zones.                                      
+      
+         * A regional cluster creates pods across zones in a region(three by default), distributing Kubernetes resources across multiple zones in the region. This is different from the default cluster, which has all its resources within a single zone(as shown above).
+         
+         * A regional cluster is also highly available(HA) with zero downtime for upgrades. This results in the cluster being highly available for deployments even in situations like a single node being upgraded.  
+         
+         * They also increase control plane uptime to 99.95%. 
+         
+         * To avoid tripling the number of nodes without losing these benefits, the ``--node-locations`` flag suffixed with a location can be used. 
```

Comment:
```suggestion
         * To avoid tripling the number of nodes while still having HA kubernetes, the ``--node-locations`` flag can be used to specify a single zone to use.
```